Package.swift

```diff
@@ -22,6 +22,7 @@ let package = Package(
         .package(url: "https://github.com/RLovelett/Ogra.git", .branch("master")),
         .package(url: "https://github.com/thoughtbot/Curry.git", from: "4.0.0"),
         .package(url: "https://github.com/RLovelett/swift-package-manager.git", .branch("swift-4.1-branch")),
+        .package(url: "https://github.com/tadija/AEXML.git", .branch("master"))
```

Comment:
Unless there is a particular reason not to I'd rather us lock to a version rather than a branch. Usually the only exception to that is when it is from one of my dependencies. Like I had to fork the repo to get something to work, for example swift-package-manager or Ogra.

Reply:
:+1: No reason not to pin. Pinned to 4.3.0.